src/components/TodoList/TodoList.tsx

```diff
@@ -1,12 +1,25 @@
 /* eslint-disable */
 import React from 'react';
+import { selector } from '../../app/store';
+import { useDispatch } from 'react-redux';
+import { filtredTodos } from '../../features/filter';
+import { setCurrentTodo } from '../../features/currentTodo';
 
 export const TodoList: React.FC = () => {
+  const filter = selector(state => state.filter);
+  const todos = selector(state => state.todos);
```

Comment:
The `selector` function is used incorrectly here. It should be `useSelector` from `react-redux` to access the Redux store state. Make sure to import `useSelector` and replace `selector` with `useSelector`.